canonical-websites/www.ubuntu.com
# Updated world map with real percentages

## Done

- Updated world map data with real data on /desktop/statistics
- black and linter updated formatting of related .js files

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/desktop/statistics](http://0.0.0.0:8001/desktop/statistics)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- See that the world data looks more like we think it should

## Issue / Card

## Screenshots


![image](https://user-images.githubusercontent.com/441217/47685153-625f6e00-dbcd-11e8-94da-c76dbec7f00b.png)
